src/components/TaskList/Task.test.js

```diff
@@ -0,0 +1,60 @@
+// Global imports
+import { fireEvent, render } from '@testing-library/react';
+import React from 'react';
+
+// Local imports
+import Task from './Task';
+
+describe('components', () => {
+  describe('TaskList.Task', () => {
+    it('should render a task', () => {
+      const STATE = 'complete';
+      const TASK = { name: 'taskName', firstPage: 'pageName', state: STATE };
+      const ON_CLICK = () => {};
+      const { container } = render(<Task task={TASK} onClick={ON_CLICK} />);
+      expect(container.childNodes.length).toEqual(1);
+      expect(container.childNodes[0].classList).toContain('hods-task-list__item');
+
+      const span = container.childNodes[0].childNodes[0];
+      const state = container.childNodes[0].childNodes[1];
+      expect(span.classList).toContain('hods-task-list__task-name');
+      expect(state.textContent).toEqual('Completed');
+    });
+
+    it('should render a task with inactive link if state is cannotStartYet', () => {
+      const NAME = 'taskName';
+      const STATE = 'cannotStartYet';
```

Comment:
```suggestion
      const STATE = TaskStates.TYPES.CANNOT_START_YET;
```

And the same for all the other hard-coded instances in here. 😁